spec/views/events/show.html.erb_spec.rb

```diff
@@ -54,17 +54,8 @@
     expect(rendered).to have_link(t(:details, scope: 'events.applicants_overview'))
   end
 
-  it "displays print applications button" do
```

Comment:
this is now event phase dependent and covered in a later test
(as in this test would be duplicate with the exact lines)